### PR TITLE
docs(example): add `qwik` example

### LINKED ISSUE #1626: Missing a example of usage with qwik framework
Missing a example of usage with Qwik framework

### PR DESCRIPTION
**Pull request by vasilGerginski**
Created a qwik app with npm create qwik@latest

### CODE REVIEW

File: examples/qwik/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Qwik App ⚡️
```
Comment (antfu) [inline examples/qwik/README.md:1]: Can we trim this example down and remove:

- README
- Static assets
- CSS files and use utilities instead

to keep it minimal as other examples. Thanks.

**Review by antfu — CHANGES_REQUESTED** [review]

File: examples/qwik/README.md (hunk 1 above)
Comment (vasilGerginski) [reply examples/qwik/README.md:1]: Yes, of course. :)

**Comment by chris-zhu** [comment]
As antfu said, and you haven't finished yet, please refer to other examples, to keep this example lightweight.

**Comment by vasilGerginski** [comment]
Let me know if the example is simple enough, Thanks

**Review by chris-zhu — APPROVED** [review]

**Review by antfu — APPROVED** [review]

**Comment by Leizhenpeng** [comment]
@zyyv  this qwik example is not working in `build mode` ! @vasilGerginski

**Comment by vasilGerginski** [comment]
I confirm it doesn't work

**Comment by cunzaizhuyi** [comment]
Sis is it easy to solve the problem in build mode？
I meet this problem also。
I need help。

**Comment by juzicheng** [comment]
> I confirm it doesn't work

Have you solved it yet? I am currently facing the same problem。

it working in dev mode, but not deploy

**Pull request merged**